Utils/Functions/Filesystem/Files/script.js

```diff
@@ -211,29 +210,15 @@ module.Checksum = class Checksum {
      * @returns {string} The calculated checksum
      */
     get() {
+        let progressBar;
         if (this._wizard) {
-            var progressBar = this._wizard.progressBar(tr("Checking file consistency..."));
```

Comment:
The previous call led to `progressBar` being available only inside the if block but not afterwards